### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by beno11224**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,35 +1,83 @@
...
+
+### Running Tests
+For every commit CircleCI runs all the included tests, however we recomend that you run test locally too before committing.
+
+####Running tests locally
+We have included a handy test configuration that can be ran from intellij.
+![Running tests locally in intellij](https://thumbs.gfycat.com/SentimentalGargantuanAmericanshorthair-size_restricted.gif)
+
+### Adding Tests
+- Create new class for tests under `/game/tests/src` When naming the class end the name with `UnitTests` for consistency e.g. `PlayerUnitTests`
```
Comment (jasonmash) [inline README.md]: This is no longer accurate

File: README.md [hunk 2]
```diff
@@ -1,35 +1,83 @@
...
+- See this page for examples of assertions: https://github.com/junit-team/junit4/wiki/assertions
+- Run your tests locally and see if they pass!
+
+### CircleCI Test Results
+After tests have run the results are displayed in the "Test Summary" tab on CircleCI.
+
+If the tests have failed and no test summary is provided, this normally means that the code doesn't compile, or there is a problem with the test code. To gather more information, scroll down to read the console output from when the tests were run.
 
-##Tech Support
+## Code layout and seperation of concerns
```
Comment (jasonmash) [inline README.md]: Can we just delete this section?

File: README.md [hunk 3]
```diff
@@ -1,35 +1,77 @@
...
 
-A specialised content management system provides for simplistic automated generation and maintainability of the website, disentangling the complexities of code documentation maintenance and assisting in the accountability and reporting of the project as a whole.
+## Testing
+This project is tested using JUnit. Tests are located within the `/game/src/tests` directory. For test documentation, please see https://github.com/junit-team/junit4/wiki
+
+### Running Tests
+For every commit CircleCI runs all the included tests, however we recomend that you run test locally too before committing.
+
+### Adding Tests
+- Create new class for tests under `/game/tests/src` When naming the class end the name with `_Test` for consistency e.g. `Player_Test`
```
Comment (beno11224) [inline README.md]: do i give the full filepath?

Comment (jasonmash) [inline README.md]: that's fine

File: README.md (hunk 2 above)
Comment (beno11224) [inline README.md]: i mean we can, i just though it was nice

**Pull request merged**